### PR TITLE
add: x漫画网站(https://xmanhua.com)，指定漫画更新提醒

### PR DESCRIPTION
**Pull request by Yemaoxin**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/xmanhua/73xm
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/xmanhua.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (DIYgod) [inline docs/xmanhua.md:1]: Please use right docs file

**Review by DIYgod — CHANGES_REQUESTED** [review]

**Comment by Yemaoxin** [comment]
I has deleted the wrong md file, and add the content to docs/anime.md

File: lib/routes/xmanhua/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    } else {
+        description_ = '连载中';
+    }
+    ctx.state.data = {
+        title: `x漫画  ${name}`,
+        link: `https://xmanhua.com/${uid}`,
+        description: description_,
+        allowEmpty: true,
+        item: items,
+    };
```
Comment (SukkaW) [inline lib/routes/xmanhua/index.js:72]:
```suggestion
    const description = finished ? '已完结' : '连载中';

    ctx.state.data = {
        title: `x漫画  ${name}`,
        link: `https://xmanhua.com/${uid}`,
        description,
        allowEmpty: true,
        item: items,
    };
```

File: lib/routes/xmanhua/index.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    const data = response.data;
+    const $ = cheerio.load(data);
+    const list = $('div #chapterlistload').find('.detail-list-form-item');
+    // 作者
+    const autherName = $('body > div.detail-info-1 > div > div > p.detail-info-tip > span:nth-child(1)').text().split('：')[1];
+    // 检查漫画是否已经完结
+    const finished_text = $('div.detail-list-form-title').clone().children().remove().end().text();
+    let finished = false;
+    let newOneDate = finished_text.split(',')[1];
+    if (newOneDate.search('月') !== -1 && newOneDate.search('號') !== -1) {
```
Comment (SukkaW) [inline lib/routes/xmanhua/index.js:25]:
```suggestion
    if (newOneDate.includes('月') && newOneDate.includes('號')) {
```

File: lib/routes/xmanhua/index.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    const uid = ctx.params.uid;
+    const host = 'https://xmanhua.com';
+    const url_t = `https://xmanhua.com/${uid}/`;
+    const response = await got({
+        method: 'get',
+        url: url_t,
+        headers: {
+            Referer: host,
+        },
+    });
```
Comment (SukkaW) [inline lib/routes/xmanhua/index.js:14]:
```suggestion
    const url = `https://xmanhua.com/${uid}/`;
    const response = await got({
        method: 'get',
        url,
        headers: {
            Referer: host,
        },
    });
```

File: lib/routes/xmanhua/index.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    if (newOneDate.search('月') !== -1 && newOneDate.search('號') !== -1) {
+        const month = parseInt(newOneDate.split('月')[0]);
+        const date = parseInt(newOneDate.split('月')[1].split('號')[0]);
+        const year = new Date().getFullYear();
+        newOneDate = new Date(year, month - 1, date + 1);
+    } else {
+        newOneDate = new Date(newOneDate);
+        newOneDate.setDate(newOneDate.getDate() + 1);
+    }
+    if (finished_text.search('已完結') !== -1 || finished_text.search('已完结') !== -1) {
```
Comment (SukkaW) [inline lib/routes/xmanhua/index.js:34]:
```suggestion
    if (finished_text.includes('已完結') || finished_text.includes('已完结')) {
```

**Comment by Yemaoxin** [comment]
I has accepted your suggestions and updated the codes

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please use v2 script standard https://github.com/DIYgod/RSSHub/discussions/8236

Please use parseDate() from `@/utils/parse-date` for date handling
https://docs.rsshub.app/joinus/pub-date.html

**Comment by TonyRL** [comment]
Migrated to v2
Removed pubDate for items that have no date. See: https://docs.rsshub.app/joinus/pub-date.html#mei-you-ri-qi

**Pull request merged**